### DISCUSSION TITLE
Whitelist Source IP for Istio NLB

### DISCUSSION THREAD

**Discussion by nirajdesai2909** [istio/istio#49171] [Q&A]
Does anyone know how can i whitelist source IP for istio <> NLB

My AWS NLB does not have security groups and i can see the loadbalancersourceranges configuration in my service manifest. I tried to change it , but unfortunately that is not working for me.

Any leads on above?

**Comment by husnialhamdani** [comment]
Is this what you need https://istio.io/latest/docs/tasks/security/authorization/authz-ingress/#ip-based-allow-list-and-deny-list?

**Reply by nirajdesai2909** [reply]
Yeah i tired the above approach but it does not seem to work for me , I added the right app labels as well.

When i specify 0.0.0.0/0 (ALL) it works and when i specify certain IPs that i need does not work

**Reply by nirajdesai2909** [reply]
I AM GETTING RBAC ACCESS DENIED  error for the authorization policies

**Reply by zirain** [reply]
related to https://istio.io/latest/docs/ops/configuration/traffic-management/network-topologies/#configuring-network-topologies?

**Reply by husnialhamdani** [reply]
I guess so, to preserve the actual source IP and then we can use the AuthZ policy.

**Comment by rootsongjc** [comment]
I think first you need to get Istio to be able to get the real source IP, I think this article can help: https://tetrate.io/blog/istio-source-ip-transparency/ @nirajdesai2909
